wordpress-mobile/WordPress-iOS
# Update pods

Updated pods and the version of Cocoapods we use to `1.4.0`.

Output of `pod outdated` is now:

```
The following pod updates are available:
- CocoaLumberjack 3.2.1 -> 3.2.1 (latest version 3.4.1)
- lottie-ios 1.5.1 -> 1.5.1 (latest version 2.5.0)
- UIDeviceIdentifier 0.5.0 -> 0.5.0 (latest version 1.1.1)
```

The AFNetworking update introduced some new warnings to the project, let me know if you want to keep it that way or roll it back?

<img width="453" alt="screen shot 2018-02-20 at 18 30 51" src="https://user-images.githubusercontent.com/1594081/36439309-c871764c-166c-11e8-9a96-26d4480faf8e.png">

Fixes #8609.

To test:
Run the app and make sure nothing obvious breaks. I clicked around the app, tried logging in with Google and everything seemed fine.
